backend-opal/src/server/portfolio.js

```diff
@@ -0,0 +1,34 @@
+const portfolioList = [
+    {id: '1', userId: '1', symbol: 'MSFT', amount: 123}
+]
+
+exports.getPortfolio = function (server) {
+    server.get('/portfolio', function (req, res) {
+        let authorization = req.headers.authorization;
+        if (!authorization || !authorization.startsWith('Bearer')) {
+            var response = {
+                "error": 'Authentication header is missing!'
+            };
+
+            res.statusCode = 401;
+            res.setHeader('Content-Type', 'application/json');
+            res.end(JSON.stringify(response));
```

Comment:
@WenyuD this is also more convenient with express `res.send(anyObject)` will set the header and stringify the content